packages/render-fidelity-tools/src/common.ts

```diff
@@ -25,19 +25,19 @@ export const COMPONENTS_PER_PIXEL: number = 4;
 export const MAX_COLOR_DISTANCE: number = 35215;
 
 export interface ImageComparisonAnalysis {
-  matchingRatio: number;
-  averageDistanceRatio: number;
-  mismatchingAverageDistanceRatio: number;
   rmsDistanceRatio: number;
 }
 
 export interface ImageComparisonResults {
-  analysis: ImageComparisonAnalysis;
+  analysis: ImageComparisonAnalysis
+}
+
+export interface Visuals {
   imageBuffers: {delta: ArrayBuffer|null; blackWhite: ArrayBuffer | null;};
 }
 
 export interface ScenarioRecord {
-  analysisResults: Array<Array<ImageComparisonAnalysis>>;
```

Comment:
the analysisResult used to be an array of ImageComparisonAnalysis calculated in many thresholds. Now i removed all thresholds based metrics, so it should be only one ImageComparisonAnalysis with only a rms in it.